Example/PrimerSDK/MerchantCheckoutViewController.swift

```diff
@@ -20,21 +20,45 @@ class MerchantCheckoutViewController: UIViewController {
     let endpoint = "https://us-central1-primerdemo-8741b.cloudfunctions.net"
     let amount = 200
     
+    let bgThread1 = DispatchQueue(label: "test1", qos: .background, attributes: .concurrent, autoreleaseFrequency: .workItem, target: nil)
```

Comment:
in terms of DX, will we require any developer to define these threads when implementing Primer?

Reply:
No, I have to remove them. Good catch. This was only for me testing calling each of our functions on a completely different thread.